action.py

```diff
@@ -255,7 +255,33 @@ def get_or_create_beanstalk_application_version(
     )
 
 
-def main(config: Config):
+def check_aws_credentials():
+    aws_credential_variables = ("AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY")
+    if not all(variable in os.environ for variable in aws_credential_variables):
+        raise ValueError(f"AWS credentials not configured ({', '.join(aws_credential_variables)})")
```

Comment:
One could output only the missing variables, but this is probably clear as well.

Reply:
Ideally, sure - but I think it's too much work and isn't worth it.